package.json

```diff
@@ -70,11 +70,13 @@
     "fetch-ponyfill": "^6.0.2",
     "sturdy-websocket": "^0.2.1",
     "tslib": "^2.1.0",
-    "web3": "1.4.0",
-    "web3-core": "1.4.0",
-    "web3-core-subscriptions": "1.4.0",
-    "web3-eth": "1.4.0",
-    "web3-utils": "1.4.0",
+    "web3": "1.5.0",
+    "web3-core": "1.5.0",
+    "web3-core-helpers": "^1.5.0",
+    "web3-core-method": "^1.5.0",
```

Comment:
Use pinned versions for all the web3 versions (no `^` in the version). We want to make sure that all the web3 packages are on the same version, and we also want it pinned because alchemy-web3 uses internal web3 packages and there's no guarantee they will stay the same between versions.

(Re-run `yarn` after changing `package.json` to update the lockfile.)